heron/instance/tests/java/com/twitter/heron/instance/spout/SpoutInstanceTest.java

```diff
@@ -70,106 +63,51 @@
  */
 public class SpoutInstanceTest {
   private static final String SPOUT_INSTANCE_ID = "spout-id";
-  private static IPluggableSerializer serializer;
-  private WakeableLooper testLooper;
-  private SlaveLooper slaveLooper;
+  private static IPluggableSerializer serializer = new JavaSerializer();
 
   // Singleton to be changed globally for testing
   private AtomicInteger ackCount;
   private AtomicInteger failCount;
   private PhysicalPlans.PhysicalPlan physicalPlan;
```

Comment:
for the `physicalPlan` seems each test has its own value, can we make it a local variable?